### ISSUE TITLE
text cut off in search

### ISSUE THREAD

**Issue by 51k34** [davidtakac/bura#35]
Can be seen with the letters y, q, p and j

![image](https://github.com/davidtakac/bura/assets/92328005/36cc3a91-efda-425a-9339-d292a3da7a95)
![image](https://github.com/davidtakac/bura/assets/92328005/848b45be-08fd-472a-aaf0-b61ce273dac8)

**Comment by davidtakac** [comment]
Yes. This is an unfortunate bug in Google's Material 3 library for Jetpack Compose. There's a few other rough edges I'm waiting for them to fix with the Search Bar component. This is a good reminder for me to create an issue in their issue tracker so it gets addressed. Thanks!

**Comment by davidtakac** [comment]
They fixed this, will be in the next build
